### DISCUSSION TITLE
Is it possible to convert this command to magick.net?

### DISCUSSION THREAD

**Discussion by spinningcat** [dlemstra/Magick.NET#602] [General]
Hello everyone 

i want to convert this command to magick.net. Is it possible?

identify -verbose -features 1 -moments -unique kitchen\ \(1\).jpg

Sincerely
Thanks

**Comment by dlemstra** [comment]
You can get the `-moments` with the `Moments()` method of `MagickImage`. And `-unique` with the `UniqueColors()` method. And the `Statistics()` method can be used to get the information from `-features 1`.
